lib/engine/game/g_1822_pnw/game.rb

```diff
@@ -604,6 +656,25 @@ def regional_railway?(entity)
           @regional_railways ||= %w[A B C].freeze
           @regional_railways.include?(entity.id)
         end
+
+        def associated_minors
+          @corporations.select { |c| c.floated? && MINOR_ASSOCIATIONS.include?(c.id) }
+        end
+
+        def unassociated_minors
+          @corporations.select { |c| c.floated? && c.type == :minor && !MINOR_ASSOCIATIONS.include?(c.id) }
+        end
+
+        def associated_major(minor)
+          corporation_by_id(MINOR_ASSOCIATIONS[minor.id])
+        end
+
+        def forest?(tile)
+          tile.parts.each do |part|
+            return true if part.is_a?(Engine::Part::Upgrade) && (part.terrains[0] == :forest)
```

Comment:
What do you think about associating a forest icon instead of using terrain? It might be more straight-forward.

Reply:
I went with terrain because those hexes have a terrain cost of $10 when you put a tile on them - so it seemed more appropriate.

Reply:
Okay, if I understand. The forest terrain informs which tiles the lumber ability can be placed on. Once placed, a forest icon is assigned to the tile. Is that correct?

Tile has a method called terrain. Why doesn't that method work for you?

Reply:
Because I didn't see it. :)

Changed to:
          tile.terrain.include?(:forest)